### DISCUSSION TITLE
I cant set defaultValue as ""

### DISCUSSION THREAD

**Discussion by nani1008** [react-hook-form/react-hook-form#4851] [Q&A]
So I started migrating my v6.0 to v7.0 and i'm using this to handle my defaultValues but in v7.0 it's not working how can I achieve this? 

> const {
>     register,
>     formState: { errors },
>     handleSubmit,
>   } = useForm({
>   defaultValues: {
>       password: "",
>       firstName: "",
>     },
>   });

**Comment by nani1008** [comment]
I saw the issue regarding this I cant use it with my custom input component? Is this a bug? Thank you

**Reply by bluebill1049** [reply]
share a CSB
